RepoDb/RepoDb/ClassMappedNameCache.cs

```diff
@@ -44,7 +44,7 @@ public static string Get<TEntity>(bool quoted = true)
         /// <returns>The cached mapped name of the entity.</returns>
         internal static string Get(Type type, bool quoted = true)
         {
-            var key = type.FullName.GetHashCode() + quoted.GetHashCode();
+            var key = type.FullName.GetHashCode() ^ quoted.GetHashCode();
```

Comment:
@calumchisholm - Thank you for reverting back with this solution. The first time I tested this one, it failed and that resulted in a collision. But, of course, I will retest this one again soon on my development environment and might merge this to the main branch.